Packages/macOS/CmuxTerminal/Tests/GhosttyRuntimeTestStubs/GhosttyRuntimeTestStubs.c

```
@@ -23,7 +23,11 @@ bool ghostty_surface_clear_selection(void *surface) {
 }
 
 void ghostty_config_diagnostics_count(void) {}
+void ghostty_config_free(void) {}
```

Cppcheck (2.21.0)

[style] The function 'ghostty_config_free' is never used.

(unusedFunction)